website/config/routes.js

```diff
@@ -481,7 +481,9 @@ module.exports.routes = {
   'GET /docs/using-fleet/standard-query-library': (req,res)=> { return res.redirect(301, '/guides/standard-query-library');},
   'GET /docs/using-fleet/mdm-commands': (req,res)=> { return res.redirect(301, '/guides/mdm-commands');},
   'GET /docs/using-fleet/log-destinations': (req,res)=> { return res.redirect(301, '/guides/log-destinations');},
+  'GET /articles/how-to-uninstall-osquery': (req,res)=> { return res.redirect(301, 'articles/how-to-uninstall-fleetd');},
 
+  
```

Comment:
This redirect will need to go to /guides/how-to-uninstall-fleetd

```suggestion
  'GET /guides/how-to-uninstall-osquery': (req,res)=> { return res.redirect(301, '/guides/how-to-uninstall-fleetd');},

```

Reply:
Thanks @spokanemac & @eashaw!

I knew there had to be more to this type of change but was unsure of how or where. Thanks for making the changes. I'll do my best to remember this and reference how you made the changes in future PRs.